src/main/groovy/com/netgrif/application/engine/petrinet/domain/dataset/logic/action/ActionDelegate.groovy

```diff
@@ -218,39 +227,121 @@ class ActionDelegate {
     }
 
     /**
-     * Changes behavior of a given field on given transition if certain condition is being met.
+     * Changes behavior of a given field on given transition if certain condition is being met or on each transition containing a field.
      * <br>
-     * Example:
+     * Example 1:
      * <pre>
      *     condition: f.conditionId,
      *     text: f.textId,
      *     transition: t.transitionId;
      *
      *     make text,visible on transition when { condition.value == true }* </pre>
-     * This code will change the field <i>text</i> behaviour to <i>visible</i> when fields <i>condition</i> value is equal to <i>true</i>
+     * This code will change the field <i>text</i> behaviour to <i>visible</i> when fields <i>condition</i> value is equal to <i>true</i>.
+     *
+     * Example 2:
+     * <pre>
+     *      text: f.textId;
+     *
+     *      if (condition)
+     *          make text, visible on each
+     * </pre>
+     * This code will change the field <i>text</i> behaviour to <i>visible</i> on each transition that contains the field <i>text</i>.
      * @param field which behaviour will be changed
-     * @param behavior one of visible, editable, required, optional, hidden, forbidden
+     * @param behavior one of initial, visible, editable, required, optional, hidden, forbidden
      */
     def make(Field field, Closure behavior) {
```

Comment:
Implement these `make` variants (f1,f2 = data field; t1,t2 = transitions):
make f1, visible on t1 when { }
make [f1,f2], visible on t1 when { }
make f1, visible on [t1,t2] when { }
make f1, visible on transitions when { }

`transitions` is keyword for all transitions where the data field is referenced (dataRef for the field exists in transition). 
`when(Closure)` is optional in all variants of the statement.

Reply:
new variants created